babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  'plugins': ['@babel/plugin-transform-modules-commonjs'],
```

Comment:
can you write a paragraph explaining why we need to add Babel to the ever-growing-list-of-compilers we need to support? Looks like it's only used by Jest?

Reply:
hang on I just realized this isn't `studio-internal` 😆 um... I'm still curious though

Reply:
It's because of of `lodash-es`. I suspect I should be able to instrument TS to transpile those files, though, so will give it a try.

Reply:
I see we replaced `lodash` - what is `lodash-es` doing for us?

Reply:
Rollup does not really like CJS modules, so this is the primary reason.
Lodash-es is mostly used in a worker code, so it does not really change much now.